pkg/hintrunner/zero/zerohint_math.go

```diff
@@ -1189,3 +1189,46 @@ func createIsQuadResidueHinter(resolver hintReferenceResolver) (hinter.Hinter, e
 
 	return newIsQuadResidueHint(x, y), nil
 }
+
```

Comment:
can you add a comment that describes the hint here? you can take inspiration from other hints (for formatting)